corehq/apps/app_manager/static/app_manager/js/manage_releases_by_app_profile.js

```diff
@@ -12,7 +12,6 @@ hqDefine('app_manager/js/manage_releases_by_app_profile', [
     _,
     initialPageData,
     assertProperties,
-    toggles
```

Comment:
you should also remove the dependency from line 7

Reply:
Done!